src/main/java/se/diabol/jenkins/pipeline/domain/Component.java

```diff
@@ -31,19 +36,34 @@
     private final String firstJob;
     private final String firstJobUrl;
     private final boolean firstJobParameterized;
+    
+    private final int noOfPipelines;
+    
+    private int componentNumber = 0;
+    
+    private String pagingData = null;
+    
+    private boolean enablePaging = false;
```

Comment:
Same as in DeliveryPipelineView; Could you please rename it to pagingEnabled as it reads better?